webcompat/static/js/lib/bugform.js

```diff
@@ -227,6 +232,29 @@ function BugForm() {
     }
   };
 
+  this.clearStepsToReproducePlaceholder = function() {
+    var stepsToReproduceStrAr = this.stepsToReproduceField
+      .val()
+      .split("4. Nothing happened.");
+
+    if (stepsToReproduceStrAr.length > 1 && stepsToReproduceStrAr[1] === "\n") {
```

Comment:
hi @miketaylr I basically agree with you here. I wasn't totally clear about the logic except I did trial/error logging to find out what worked. maybe it had something to do with using that URLSearchParams in the Nightly change (?)... but I tried to clean up the logic and moved the logic that adds the `details` param to the text area in the onFormSubmit(). there is just a little residual logic in the text area focus handler I needed to retain in order to handle the case where the user had previously edited the field. other changes from Friday's comments were also pushed.